### ISSUE TITLE
Consistent heights for cards

### ISSUE THREAD

**Issue by mrjmd** [contentacms/contenta_angular#56]
Sometimes our recipe list cards line up and look perfect:

![image](https://user-images.githubusercontent.com/2111857/28492476-8e3c9408-6eb9-11e7-83ad-88145ff81f0c.png)

But oftentimes the heights are off:

![image](https://user-images.githubusercontent.com/2111857/28492480-9c47ddb4-6eb9-11e7-8569-91842173d678.png)

Even if the images / text length isn't quite the same, let's keep the height of the overall card consistent.

**Comment by joaogarin** [comment]
The problem here sounds like its the missing "difficulty" in that middle recipe..I don't know if thats a bug on the backend. Fixing every height is also not a great solution.

Right now there are two main problems around this :

1 ) Super inconsistent thumbnails. This has been discussed in the main channel, as soon as we start using proper images and use image cache this should go away. We are mitigating this with what to me is a super hack..but at least makes it look ok.

2 ) Sometimes missing information comes..and we have to figure out if that is normal or if its a bug on a specific recipe (no title, no tag, no time for example..we need to figure out the possible variations of a card basically)

I will check in the main channel about these two topics. Cause having to make a fix height for tags, card, images etc..makes it really "hardcoded" which is not great.

**Comment by mrjmd** [comment]
Yeah, my screenshots were a bad example but it's definitely happening on some rows without the "difficulty" problem.
